src/services/item/repository.test.ts

```diff
@@ -729,6 +730,69 @@ describe('ItemRepository', () => {
       });
     });
   });
+  describe('postMany', () => {
+    it('post many', async () => {
+      const items: Item[] = [];
+      for (let i = 0; i < 15; i++) {
+        const newItem = itemRepository.createOne({
+          name: `item${i}`,
+          type: ItemType.FOLDER,
+          creator: actor,
+        });
+        items.push(newItem);
+      }
+
+      const insertedItems = await itemRepository.addMany(items, actor);
+      const insertedItemNames = insertedItems.map((i) => i.name);
+      const insertedItemTypes = insertedItems.map((i) => i.type);
+      const insertedItemCreatorIds = insertedItems.map((i) => i.creator?.id);
+
+      const itemsInDB = await testUtils.rawItemRepository.find({
+        where: { name: In(insertedItemNames) },
+        relations: { creator: true },
+      });
+      const itemNamesInDB = itemsInDB.map((i) => i.name);
+      const itemTypesInDB = insertedItems.map((i) => i.type);
+      const itemCreatorIdsInDB = insertedItems.map((i) => i.creator?.id);
+
+      expect(itemNamesInDB.sort()).toEqual(insertedItemNames.sort());
+      expect(itemTypesInDB.sort()).toEqual(insertedItemTypes.sort());
+      expect(itemCreatorIdsInDB.sort()).toEqual(insertedItemCreatorIds.sort());
```

Comment:
I would have an additional check on the path, that is supposed to be a root path.